docs/API.md

```diff
@@ -176,7 +176,7 @@ query HelloWorld {
 | Option          | Default                                          | Description                                                            |
 |-----------------|--------------------------------------------------|------------------------------------------------------------------------|
 | service         | *Service name of the app suffixed with -graphql* | The service name for this integration.                                 |
-| filterVariables | `undefined` *No variables will be recorded*      | To enable recording provide a callback. E.g. `variables => variables`. |
+| variables       | `undefined` *No variables will be recorded*      | To enable recording provide a callback. E.g. `variables => variables`. |
```

Comment:
I’m not sure what you mean by the default being `{}`. If no callback is provided then no variables are recorded. Maybe before recording these variables separately as tags I was defaulting to an empty object?

For now I have only updated the description and left the default value.

Reply:
What I mean is that the callback returns an object, so the default is technically an empty object meaning no variable will get recorded (same as if returning an empty object). I'm trying to find a way to avoid having to mention "No variables will be recorded" basically. I guess we can iterate if someone ever gets confused :)

Reply:
Ah. I guess I approached it from the perspective of making it very explicit to the reader that we won’t record them by default, which could be something that worries them.